### ISSUE TITLE
General questions regarding PS3

### ISSUE THREAD

**Issue by boseongyun** [macss-modeling/General-Questions#19]
Hello! 

I am writing to ask a few quick questions regarding the hyperparameters that we have to tune for this assignment. I have,

* Decision Tree: cost, tree depth, and minimum observations
* Bagging: cost, tree depth, and minimum observations
* Random Forest: mtry, tress, and minimum observations
* Graident Boosting: tree, depth, mtry, learn rate, loss reduction, minimum observations, and many others. 

I am sorry for providing a lengthy list of the parameters but could it be possible to get information about what to tune for in each model? 

Thank you very much for your time!

**Comment by pdwaggoner** [comment]
Thanks for the question @boseongyun . Tuning is entirely up to you. I know that's likely frustratingly ambiguous, but this is why we started Tuesday's class showing the random forest with the *default* values for all hyperparameters, and how this model performed quite closely to the "tuned" version. So, feel free to make a call as to how you think best to tune, and of course if you're unsure, do it both ways and show results side by side as we did in class (e.g., plotting the version with default (no tuning) and the version with tuning all hyperparameters, and discussing differences if any). Importantly, though, with XGBoost, many of those hyperparameters don't have default values, so you'll have to read the documentation for which ever version you decide to use (e.g., scikit-learn, tidymodels, caret, xgboost package, etc.). But you have the starter code from our in-class session for XGB to get you started.
